services/tenant-ui/frontend/src/components/issuance/credentialtemplate/CreateCredentialTemplate.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+    <Button
+        icon="pi pi-id-card" 
+        class="p-button-text"        
+        @click="displayForm = true"
+        v-tooltip.top="'Create Credential Template'"
+    />
+    <Dialog header="Create Credential Template" v-model:visible="displayForm">
+        <CreateCredentialTemplateForm 
+            @form-close="displayForm = false;"
+            :schema-template-id="schemaTemplateId"
+        />
+    </Dialog>
+</template>
+<script setup lang="ts">
```

Comment:
Annoying style quibble but I'd love to keep a space between template, script, and style blocks